go.mod

```diff
@@ -3,12 +3,12 @@ module github.com/chewong/rg-cleanup
 go 1.13
 
 require (
-	github.com/Azure/azure-sdk-for-go v36.2.0+incompatible
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.6.0
-	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.3.0
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.15.0
```

Comment:
We should take a pass at updating these dependencies sometime, but these versions have been working for me locally for now.